### PR TITLE
fix: split vehicle model package into `description` and `launch`

### PR DESCRIPTION
**Pull request by kenji-miyake**
## PR Type

- Fix for API change

## Related Links

https://github.com/tier4/lexus_description.iv/pull/23

## Description

Fix the package path for https://github.com/tier4/lexus_description.iv/pull/23.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: vehicle_launch/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ ex.)
 <https://github.com/tier4/autoware_launcher/blob/d880474/vehicle_launch/launch/vehicle_interface.launch.xml#L9>
 
 ```xml
-<let name="vehicle_model_pkg" value="$(find-pkg-share $(var vehicle_model)_description)"/>
+<let name="vehicle_launch_pkg" value="$(find-pkg-share $(var vehicle_launch)_launch)"/>
```
Comment (wep21) [inline vehicle_launch/README.md]:
```suggestion
<let name="vehicle_launch_pkg" value="$(find-pkg-share $(var vehicle_model)_launch)"/>
```

Comment (kenji-miyake) [reply vehicle_launch/README.md]: Oh... nice catch. Thank you!

**Comment by yukkysaito** [comment]
LGTM

**Review by yukkysaito — APPROVED** [review]

**Comment by h-ohta** [comment]
@kenji-miyake When dou you merge this pr?

**Comment by kenji-miyake** [comment]
@h-ohta Any time is okay for me, I think it's up to the UC team.
Do you have any requests?

**Comment by h-ohta** [comment]
@kenji-miyake I got it. I also created PR for my vehicle. So I'd like to adjust the merge timing.

**Comment by kenji-miyake** [comment]
@h-ohta It's great! But we have to care about other vehicles as well.

**Comment by takayuki5168** [comment]
@kenji-miyake Why was this PR closed?
You tried to make the directory structure of tier4's and awf's vehicle description/launch the same, right? I also want to realize it.

**Comment by kenji-miyake** [comment]
@takayuki5168 Just because there were no comments from integration team members.
I can reopen it if you want, I have no resources to follow-up this PR. If you can take leadership, I really appreciate it.

**Pull request closed without merging**